.github/workflows/ci.yml

```diff
@@ -367,7 +377,8 @@ jobs:
       - name: Use Node.js latest
         uses: actions/setup-node@v3
         with:
-          node-version: "*"
+          node-version: latest
+          check-latest: true
```

Comment:
The runtime integration test has been failing since Node.js 17+. However we didn't caught the error because `setup-node` used the latest _cached_ node versions, i.e. Node 16.x.